apps/dialtone-documentation/docs/.vuepress/theme/components/Page.vue

```diff
@@ -101,4 +102,9 @@ const includeToc = computed(() => {
   const headers = usePageData().value.headers;
   return headers?.length > 0;
 });
+
+const tokensPageClass = computed(() => {
+  if (route.path.includes('tokens')) return 'tokens-page';
+  return '';
```

Comment:
Im guessing this will be escalable soon.

We will think to have this dynamically sth like `pageDynamicallyClasses`
Then it will handle these classes.

Reply:
We can think about that in the case that some other page needs a custom layout like this one